01_01_Basic_Rules.txt

```diff
@@ -0,0 +1,1311 @@
+The Primary Game Mechanic
+==============================
+
+	You may now be thinking to yourself; "Self-person, we sure can't wait to go
+gallivanting through the galaxy doing fantastic things and seeing the beautiful
+galaxies! But how would we do that?" Well, yourself doesn't have the answer yet,
+but good news! You just found it!
+
+	In Compound X, most things you will do will be accompanied by a 'roll'. That 
+translates to rolling the dice. Rolling the dice is our way of factoring in
+the human (or not human) tendency to do things in a way which isn't perfect.
+When you roll high, you do very well. When you roll low, you goof up. But hold 
+your horses, or kangaroo or what have you. This is a strategy RPG, not a gambling game.
+Each roll will have numbers which you will add to or subtract from the one you 
+get from the dice. This represents how good or bad your particular character is
+at doing whatever it is you are attempting to do. What to add when is discussed
+in detail below. 
+
+	For now, know that Compound X uses two ten-sided dice (usually called D10s) to
+represent a d100, a percent. One of the D10s represents the number in the 10's place.
+the other represents the number in the 1's place. Most dice sets come with two
+D10s, one with a zero after the number on the face, the other without that zero.
+In that case, obviously you add the numbers on the face of the dice together to
+find out what you rolled. If your dice came from different sets, you must pick
+which dice represents the tens place before you roll. i.e. if one die rolls a
+40 and the other rolls a 6, you rolled 46%. Alternatively, if you have an orange
+die which represents your ten's place and a purple die which represents your one's
+place, and you roll an orange 3 and a purple 9 then you rolled 39%.
+
+	It's important to note that you don't have to roll for everything. Your 
+character probably knows how to do mundane things like brush their teeth, open 
+an unlocked door, and tie their shoes. It's okay to just do these things, even 
+in combat.
+
+Rolling with Advantage:
+	Sometimes circumstances or the occasional feat may prompt you to make a roll 
+with Advantage or Disadvantage. In such cases you make the roll twice. With 
+Advantage you take the better roll, while with Disadvantage you take the worse 
+roll. Advantage and Disadvantage do not stack with themselves. If you would have 
+both Advantage and Disadvantage on the same roll, then they cancel each other, 
+and you just roll once.
+
+Rolling the Average:
+	There are certain circumstances and feats which allow for Rolling the Average.
+This must be announced before the die are cast. Rolling the average will give you 
+the advantage of avoiding a critical fail, but also prevents you from critically 
+succeeding. This is best used in situations when massive failure is not an option. To do
+this, you roll 2d10s, add them together, and divide by 2. The resulting number is your

```

Comment:
What's all this about "Rolling the Average"? 
I've never heard of this being used before. Is there a link to the thought behind this in github?

Anyways, rolling 2d10 divide by 2, i.e, (5+7)/2 = 6... 6%? This would be a worthless roll in a percentile system. I assume my interpretation can't be true. What does this mean?

Reply:
@Turtlelord26 ?

Reply:
It first appeared in a Rev PR for racial trait purposes. I think at the time I read it as roll the check twice and average.

Reply:
Is it actually being used for anything that anyone knows of? Just curious.

Reply:
But this can be worked on another time. # nonblocking

Reply:
Might I suggest picking this chat up in slack? https://compoundx.slack.com/messages/G8VFDBWET/

Reply:
I can't actually use slack on my laptop right now, slack won't load. So it's a bit of a pain.